refinery/ui/source/js/data-set-import/services/file-upload-status-service.js

```diff
@@ -13,9 +13,9 @@ function fileUploadStatusService () {
   vm.setFileUploadStatus = function (status) {
     var statusOptions = ['none', 'queuing', 'running'];
     if (statusOptions.indexOf(status) > -1) {
-      vm.fileUploadStatus = status;
+      vm.fileUploadStatus.status = status;
     }
-    return vm.fileUploadStatus;
+    return vm.fileUploadStatus.status;
```

Comment:
This is just the input value? Does anything need the return value?

Reply:
Not necessarily (index of values). Yes.